src/main/java/RefactoringEngine.java

```diff
@@ -95,7 +81,7 @@ public void preVisit(ASTNode node) {
 		try {
 			edits.apply(document);
 		} catch (MalformedTreeException | org.eclipse.jface.text.BadLocationException e) {
-			logger.log(Level.WARNING, e.toString());
+			System.out.println(e.toString());
```

Comment:
Why remove the logger? It would be better to replace all of the `System.out.println` calls with a logger, rather than the other way around - the logger gives us better control and will allow us to turn debugging logging on and off easily without code changes.

Reply:
(changes to the logging strategy are a separate issue, though, and should be in their own PR)